src/ccl_correlation.c

```diff
@@ -71,6 +71,16 @@ static void ccl_tracer_corr_fftlog(ccl_cosmology *cosmo,
 				  ccl_f1d_extrap_const,
 				  ccl_f1d_extrap_logx_logy,
 				  status);
+  if (*status) {
```

Comment:
Can we add a test for this? I think this should get triggered if you pass an array of C_ells that is 0 everywhere and then 1 somewhere in the middle (to avoid the shortcut)

Reply:
Good idea - added.